scraper/scraper.go

```diff
@@ -52,13 +36,47 @@ func NewScraper(resolver servicediscovery.Resolver, r *repository.ErrorsReposito
 	}
 }
 
-// Scrape stuff
+func (errorAggregates errorAggregateMap) combine(serviceName string, r *repository.ErrorsRepository,
+	rp responsePayload, targetErrorsCount targetErrorsCountMap) {
+	for _, item := range rp.ErrorAggregate {
+		if existing, exists := errorAggregates[item.AggregationKey]; exists {
+			prevCount := targetErrorsCount[rp.Target][item.AggregationKey]
+			errorAggregates[item.AggregationKey] = errorAggregate{
+				TotalCount:     existing.TotalCount + (item.TotalCount - prevCount),
```

Comment:
is there a possibility that his number would become negative and subtract instead of add to the total count?

Reply:
I did some manually testing and it didn't happen. So not sure in which circumstances might happen.